app/views/notes/index.html.erb

```diff
@@ -3,7 +3,17 @@
 <h2><%= link_to "Create new note", new_note_path, :class => 'btb btn-default' %></h2>
 
 <div>
-  <% @note.each do |note| %>
+  <%= search_form_for @q do |f| %>
+    <%= f.label :title_or_text_cont %>
+      <%= f.search_field :title_or_text_cont %>
+      <%= f.submit %>
```

Comment:
Please keep same indentation. In line 7 you have 4 spaces and here you have 6 spaces.